mappings/net/minecraft/server/world/ThreadedAnvilChunkStorage.mapping

```diff
@@ -41,31 +75,31 @@ CLASS net/minecraft/class_3898 net/minecraft/server/world/ThreadedAnvilChunkStor
 	METHOD method_17217 setLevel (JILnet/minecraft/class_3193;I)Lnet/minecraft/class_3193;
 		ARG 1 pos
 		ARG 3 level
-		ARG 4 holder
-	METHOD method_17220 getRegion (Lnet/minecraft/class_1923;ILjava/util/function/IntFunction;)Ljava/util/concurrent/CompletableFuture;
+		ARG 4 chunk
+	METHOD method_17220 createChunkRegionFuture (Lnet/minecraft/class_1923;ILjava/util/function/IntFunction;)Ljava/util/concurrent/CompletableFuture;
 		ARG 1 centerChunk
 		ARG 2 margin
 		ARG 3 distanceToStatus
 	METHOD method_17226 convertToFullChunk (Lnet/minecraft/class_3193;)Ljava/util/concurrent/CompletableFuture;
-	METHOD method_17228 save (Lnet/minecraft/class_2791;)Z
+	METHOD method_17228 saveChunk (Lnet/minecraft/class_2791;)Z
 		ARG 1 chunk
 	METHOD method_17229 getRequiredStatusForGeneration (Lnet/minecraft/class_2806;I)Lnet/minecraft/class_2806;
 		ARG 1 centerChunkTargetStatus
 		ARG 2 distance
 	METHOD method_17233 tick (Ljava/util/function/BooleanSupplier;)V
 		ARG 1 shouldKeepTicking
-	METHOD method_17235 makeChunkTickable (Lnet/minecraft/class_3193;)Ljava/util/concurrent/CompletableFuture;
-		ARG 1 holder
-	METHOD method_17236 getChunk (Lnet/minecraft/class_3193;Lnet/minecraft/class_2806;)Ljava/util/concurrent/CompletableFuture;
-		ARG 1 holder
+	METHOD method_17235 createTickingFuture (Lnet/minecraft/class_3193;)Ljava/util/concurrent/CompletableFuture;
+		ARG 1 chunk
+	METHOD method_17236 createChunkFuture (Lnet/minecraft/class_3193;Lnet/minecraft/class_2806;)Ljava/util/concurrent/CompletableFuture;
+		ARG 1 chunk
 		ARG 2 requiredStatus
 	METHOD method_17241 sendWatchPackets (Lnet/minecraft/class_3222;Lnet/minecraft/class_1923;[Lnet/minecraft/class_2596;ZZ)V
 		ARG 1 player
 		ARG 2 pos
 		ARG 3 packets
 		ARG 4 withinMaxWatchDistance
 		ARG 5 withinViewDistance
-	METHOD method_17242 save (Z)V
+	METHOD method_17242 saveAllChunks (Z)V
```

Comment:
This isn’t necessarily saving all chunks- but only unloaded chunks if flush is not true